src/ansys_sphinx_theme/__init__.py

```diff
@@ -1134,11 +1132,6 @@ def setup(app: Sphinx) -> Dict:
     use_ansys_search = app.config.html_theme_options.get("use_ansys_search", True)
     if use_ansys_search:
         update_search_config(app)
-
-    # Verify that the main CSS file exists
-    if not CSS_PATH.exists():
-        raise FileNotFoundError(f"Unable to locate ansys-sphinx theme at {CSS_PATH.absolute()}")
-    app.add_css_file(str(CSS_PATH.relative_to(STATIC_PATH)))
```

Comment:
I thought that SCSS was compiling to CSS. Thus, these were still included. Is that not the case, @Revathyvenugopal162?

Reply:
Ah! I see. Thanks.

Reply:
Anyway, I see this as a good check to make sure that the CSS is shipped with the theme. As soon as one uses it, it actually checks that the file is in place.

Reply:
Sure! I plan to check this branch against one of the libraries, possibly with the dev guide, once the migration is complete.

Reply:
I’m adding this back because the style paths are being read without this function in most cases. However, I noticed that the Sphinx Gallery CSS is taking precedence over our SCSS file, possibly due to a configuration issue. When I add this, it ensures our styles take priority.